schema/RECOUNT_AND_NP_DISBURSEMENTS.json

```diff
@@ -0,0 +1,693 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/RECOUNT_AND_NP_DISBURSEMENTS.json",
+    "version": "8.3.0.1",
+    "title": "FEC Recount Account Disbursement, National Party Recount/Legal Proceedings Account Disbursements",
+    "description": "SCHEDULE B - Recount Account Disbursement (Line 29), National Party Recount/Legal Proceedings Account Disbursements (Line 29)",
```

Comment:
I see that this reflects the Spec, but it seems like it should mention the National Party HQ and Convention disbursments which are SB21Bs

Reply:
The update to the description was made both in the JSON schema file and the spec spreadsheet.